osu.Game/Overlays/Rankings/Tables/RankingsTable.cs

```diff
@@ -35,7 +36,7 @@ protected RankingsTable(int page, IReadOnlyList<TModel> rankings)
             AutoSizeAxes = Axes.Y;
 
             Padding = new MarginPadding { Horizontal = horizontal_inset };
-            RowSize = new Dimension(GridSizeMode.Absolute, row_height);
+            RowSize = new Dimension(GridSizeMode.Absolute, row_height + row_spacing);
```

Comment:
This is hacky as all heck. I know that this is all you have to work with that maybe we need framework-level changes here to allow grid row/column spacing.

Reply:
I created an issue framework-side for that (ppy/osu-framework#3330).

I know this is hacky, but I don't feel it's *that* bad - maybe it could stay as a temporary solution until we have proper framework-side support (which I don't think is gonna happen too quickly, I presume there are more important issues to work on)

Reply:
Let's get it in for now, but with sights set at potentially replacing it in the future.